advanced/Scripts/piholeDebug.sh

```diff
@@ -91,12 +89,12 @@ get_ftl_conf_value() {
     local default=$2
```

Comment:
There can be no missing values, if FTL is there it will always return something. Even in ill-fated absence of `pihole.toml` we'll get the default path being returned. We have this I'm quite many places